app/models/spree/order/checkout_decorator.rb

```diff
@@ -0,0 +1,43 @@
+module SpreeSplitPayments::OrderCheckoutDecorator
```

Comment:
@sachinvinsol create a concern for this.
Also take care of file naming convention.